src/main.c

```diff
@@ -60,6 +61,7 @@ void m1n1_main(void)
     print_info();
     wdt_disable();
     pmgr_init();
+    pcie_init();
```

Comment:
and another nit: I think this can fail silently right now. I'd either add `printf` for each possible cause in `pcie.c` or just a single `if (pcie_init() < 0) printf("...");` here

Reply:
This depends a bit on where m1n1 is going.  Adding printf calls in pcie.c is probably the easiest way out for now. That can be easily converted to returning apropriate error codes and doing something sensible with those later.

Reply:
Since the usb code prints error messages itself I made pcie do the same.